docs/tutorial/07_setting-up-the-front-end.md

```diff
@@ -0,0 +1,70 @@
+# 7. Setting up the front-end
+
+With the backend contract written and tested, the next step is to have a front end for users to interact with it. Typically this front end is a web page executing
+JavaScript. This way the user can interact with the decentralized application (dapp) and the Ethereum blockchain without having to rely on the web server. For this tutorial 
```

Comment:
I don't get the "without having to rely on the web server". What does it mean in this context?